wakuv2/waku.go

```diff
@@ -101,9 +103,9 @@ type Waku struct {
 	symKeys     map[string][]byte            // Symmetric key storage
 	keyMu       sync.RWMutex                 // Mutex associated with key stores
 
-	envelopes   map[gethcommon.Hash]*common.ReceivedMessage // Pool of envelopes currently tracked by this node
-	expirations map[uint32]mapset.Set                       // Message expiration pool
-	poolMu      sync.RWMutex                                // Mutex to sync the message and expiration pools
+	envelopes   *ttlcache.Cache[gethcommon.Hash, *common.ReceivedMessage] // Pool of envelopes currently tracked by this node
```

Comment:
Maybe it's time to rename `envelopes` to `envelopesCache`?

Reply:
Good idea!